Bottom: cf201ed5ab78513fc28b8180be332823d352668d
Top:    9b57553636a81f772a1a04a41f10623f4d223c40
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-12-27 19:31:00 +0000

Refresh of crasy-test

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index acc6259..081b63e 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -300,7 +300,7 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
     return;
 
   r_search (layer->line_tree, drawn_area, NULL, line_callback, layer);
-  r_search (layer->arc_tree, drawn_area,  NULL, arc_callback,  layer);
+  r_search (layer->arc_tree,  drawn_area, NULL, arc_callback,  layer);
   r_search (layer->text_tree, drawn_area, NULL, text_callback, layer);
 
   /* We should check for gui->gui here, but it's kinda cool seeing the
